day2/자릿수-더하기.js

```diff
@@ -0,0 +1,15 @@
+function solution(n) {
+    let str = String(n);
+    let int = str.split('');
+    let nmap = int.map(Number);
```

Comment:
```suggestion
    const str = String(n);
    const int = str.split('');
    const nmap = int.map(Number);
```
배열 선언 방식 신경쓰기, 변수명은 누가봐도 의도를 알수 있게끔 짓도록 연습해보시길 바랍니다